mappings/1.3-pre-07261249#12w30e-server.tinydiff

```diff
@@ -6046,7 +6046,7 @@ c	net/minecraft/unmapped/C_3930157
 	m	()Ljava/lang/String;	m_8387532	m_8387532	
 c	net/minecraft/unmapped/C_3940616	net/minecraft/client/render/block/entity/BlockEntityRenderDispatcher	
 	f	Ljava/util/Map;	f_5037161	renderers	
-	f	Lnet/minecraft/unmapped/C_3940616;	f_6470555	INSTANCE	
+	f	Lnet/minecraft/unmapped/C_3940616;	f_6470555	getWorld	
```

Comment:
is this a mistake? the new name is a method name given to a field

Reply:
Yeah seems that I somehow made a mistake here